morphir/runtime/test/jvm/src/org/finos/morphir/runtime/UnitTestingSpec.scala

```diff
@@ -0,0 +1,158 @@
+package org.finos.morphir.runtime
+
+import org.finos.morphir.datamodel.Util.*
+import org.finos.morphir.datamodel.*
+import org.finos.morphir.ir.Type
+import org.finos.morphir.naming.*
+import org.finos.morphir.runtime.environment.MorphirEnv
+import org.finos.morphir.testing.MorphirBaseSpec
+import zio.test.*
+import zio.test.TestAspect.{ignore, tag}
+import zio.{Console, ZIO, ZLayer}
+
+object UnitTestingSpec extends MorphirBaseSpec {
+  val paths = List(
+    os.pwd / "examples" / "morphir-elm-projects" / "unit-test-framework" / "example-project-tests" / "morphir-ir.json",
```

Comment:
https://github.com/com-lihaoyi/sourcecode?tab=readme-ov-file#overview - If we desire to support these tests in a cross platform version we can use `sourcecode.File` to help. Use that along with File ops, but I'm fine with this being in the JVM specific folder at this point (even if I don't love it).